adafruit_motorkit.py

```diff
@@ -52,10 +51,18 @@
 class MotorKit:
     """Class representing an Adafruit DC & Stepper Motor FeatherWing, Shield or Pi Hat kit.
 
-    Automatically uses the I2C bus on a Feather, Metro or Raspberry Pi if no I2C bus
-    is supplied.
+    :param int address: I2C address of PCA9685 PWM controller. Default address is ``0x60``.
+    :param busio.I2C: I2C bus object to use. If not specified, use ``board.I2C()``.
```

Comment:
Whoops, missed this the first time, but the param name `i2c` is missing

Reply:
thanks for the careful copy-editing